public/components/ctas/8.html

```diff
@@ -0,0 +1,12 @@
+<div class="bg-sky-50">
+  <div class="p-8 max-w-7xl m-auto lg:flex lg:items-center lg:justify-between sm:p-12">
```

Comment:
I don't see the point of the `max-w-7xl m-auto`, look at what was done on the other call to actions.

Can keep the `flex` classes all the way down to the `sm` breakpoint.